Source/Hammer.spoon/init.lua

```diff
@@ -0,0 +1,95 @@
+--- === Hammer ===
+---
+--- Automate some Hammerspoon configuration tasks
+---
+--- Download: [https://github.com/Hammerspoon/Spoons/raw/master/Spoons/Hammer.spoon.zip](https://github.com/Hammerspoon/Spoons/raw/master/Spoons/Hammer.spoon.zip)
+
+local obj={}
+obj.__index = obj
+
+-- Metadata
+obj.name = "Hammer"
+obj.version = "0.1"
+obj.author = "Diego Zamboni <diego@zzamboni.org>"
+obj.homepage = "https://github.com/Hammerspoon/Spoons"
+obj.license = "MIT - https://opensource.org/licenses/MIT"
+
+obj.key_reload_config = nil
+obj.key_toggle_console = nil
+obj.configFileWatcher = nil
+
+--- Hammer.auto_reload_config
+--- Variable
+--- If true, automatically reload Hammerspoon config when any `.lua` files inside `hs.configdir` changes
+obj.auto_reload_config = true
+
+--- Hammer.install_cli
+--- Variable
+--- If true, install the `hs` command line tool
+obj.install_cli = true
+
+--- Hammer:reloadConfig()
+--- Method
+--- Manually reload configuration (wrapper around `hs.reload`)
+function obj:reloadConfig(param)
```

Comment:
I'm not sure if there's any real value in wrapping `hs.reload()` with no additional logic here?